helm/templates/ingress.yaml

```diff
@@ -0,0 +1,33 @@
+{{- if ne .Values.branch "main" }}
+---
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: {{ .Release.Name }}
```

Comment:
Again thinking about convention, should this be release name prefix with another identifier

Reply:
Updated